app/src/main/kotlin/com/wire/android/ui/home/HomeViewModel.kt

```diff
@@ -42,6 +40,17 @@ class HomeViewModel @Inject constructor(
         viewModelScope.launch {
             loadUserAvatar()
         }
+        viewModelScope.launch {
+            ongoingCalls().collect {
+                if (it.isNotEmpty()) {
```

Comment:
We need also to check the `conversationId` is the same as the viewModel's `conversationId`

Reply:
Currently I would leave this like it is, as the logic behind on sending the incoming calls/incoming call to the end user will need to be reworked as it doesn't seem to be properly propagating what was meant.

for example when the same user call for the second time the call isn't being received properly.